src/modules/custom.rs

```diff
@@ -37,6 +38,12 @@ pub fn module<'a>(name: &str, context: &'a Context) -> Option<Module<'a>> {
         }
     }
 
+    if let Some(os) = config.os {
+        if os != env::consts::OS {
```

Comment:
I think it would be nice to allow setting `os` to `unix` to exclude windows and only allow unixlike systems like with `cfg!(unix)`.

Reply:
Would the following condition be sufficient ?
```rust
if os != env::consts::OS && !(os == "unix" && env::consts::OS != "windows") {
      return None;
}
```

Reply:
I would prefer `os == "unix" && !cfg!(unix)`.

Reply:
Nice, you're right, changed to 
```rust
if os != env::consts::OS && !(os == "unix" && cfg!(unix)) {
```